ConversionTool/HtmlToImage/HtmlToImageHelper.cs

```diff
@@ -0,0 +1,22 @@
+using SixLabors.ImageSharp;
+using SixLabors.ImageSharp.Formats.Png;
+using SixLabors.ImageSharp.Processing;
+using System.IO;
+
+namespace ConversionTool.HtmlToImage
+{
+    public class HtmlToImageHelper
+    {
+        public static byte[] ResizeImage(byte[] image)
```

Comment:
Y si permitimos pasar las  dimensiones deseadas por parámetro ?
```suggestion
        public static byte[] ResizeImage(byte[] image, int width, int height )
```